crates/router/src/connector/stax/transformers.rs

```diff
@@ -37,6 +38,19 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for StaxPaymentsRequest {
                     payment_method_id: Secret::new(item.get_payment_method_token()?),
                 })
             }
+            api::PaymentMethodData::BankDebit(_) => {
+                let mut pre_auth = false;
+                if let Some(enums::CaptureMethod::Manual) = item.request.capture_method {
```

Comment:
You can make use of `is_auto_capture` from utils

Reply:
Done